mythril/laser/ethereum/plugins/implementations/mutation_pruner.py

```diff
@@ -48,17 +53,21 @@ def staticcall_mutator_hook(global_state: GlobalState):
 
         @symbolic_vm.laser_hook("add_world_state")
         def world_state_filter_hook(global_state: GlobalState):
-            if And(
-                *global_state.world_state.constraints[:]
-                + [
-                    global_state.environment.callvalue
-                    > symbol_factory.BitVecVal(0, 256)
-                ]
-            ).is_false:
-                return
+
             if isinstance(
                 global_state.current_transaction, ContractCreationTransaction
             ):
                 return
+
+            try:
+                constraints = global_state.world_state.constraints + [
+                    global_state.environment.callvalue > 0
+                ]
+
+                solver.get_model(constraints)
+                return
+            except UnsatError:
+                pass
```

Comment:
```suggestion
                # callvalue is constrained to 0, therefore there is no balance based world state mutation
                pass
```

Reply:
Maybe a comment like this is useful to indicate the edge case that is introduced here